pkg/instancetype/instancetype_test.go

```diff
@@ -287,6 +339,23 @@ var _ = Describe("Instancetype and Preferences", func() {
 				Expect(*instancetypeSpec).To(Equal(f.Spec))
 			})
 
+			It("uses client when instancetype not found within informer", func() {
+				err := clusterInstancetypeInformerStore.Delete(f)
+				Expect(err).ToNot(HaveOccurred())
+
+				instancetypeSpec, err := instancetypeMethods.FindInstancetypeSpec(vm)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(*instancetypeSpec).To(Equal(f.Spec))
```

Comment:
Just noticed the inconsistent naming of instancetypes/preferences. Sometimes the variable name assigned to in BeforeEach is longer and somethings shorter (`f`). Might make sense to use a consistent way of naming?

Reply:
I honestly thought we cleaned all of these up in an older cleanup series, I can sort this out in a fup.